build.yaml

```diff
@@ -477,6 +477,18 @@ steps:
     - batch_pods_ns
     - deploy_batch
     - test_batch_image
+ - kind: deploy
+   name: batch_logs
+   namespace:
```

Comment:
What's this for? I assume it's for copying the batch logs out. Where does that happen?

Reply:
The wait will trigger a listing of the batch server logs.